File: Data Structures & Algorithms/find-duplicate-integer/submission-2.cpp
class Solution {
public:
    int findDuplicate(vector<int>& nums) {
        vector<int> check(nums.size(), 0);

        for(int i = 0; i < nums.size(); ++i){
            if(check[nums[i]] > 0){
                return nums[i];
            }else{
                ++check[nums[i]];
            }
        }

        return 0;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        res = 0

        for num in nums:
            if num-1 not in nums:
                count = 1
                while num+count in nums:
                    count += 1
                if count > res:
                    res = count

        return res
